lms/services/event.py

```diff
@@ -49,6 +49,10 @@ def _get_type_pk(self, type_: EventType.Type) -> int:
 
         return event_type.id
 
+    @classmethod
+    def from_db_session(cls, db):
```

Comment:
A small concession here for some future-proofing.

We don't use the default factory in this type of usage but the registered factory does use the new class method..